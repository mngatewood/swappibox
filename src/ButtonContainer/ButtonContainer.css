.ButtonContainer {
  display: flex;
  justify-content: space-around;
}

button {
  width: 100px;
  height: 50px;
  font-size: 18px;
}
