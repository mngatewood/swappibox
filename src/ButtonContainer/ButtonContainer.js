import React from 'react';
import './ButtonContainer.css';
import Button from '../Button/Button'

const ButtonContainer = ({favorites}) => {
  const buttonKeys = ["People", "Planets", "Vehicles"]
  const buttons = buttonKeys.map((button) => {
    return <Button 
      key={button} 
      title={button} 
    />
  })
  return (
    <div className="ButtonContainer">
      {buttons}
    </div>
  )
} 

export default ButtonContainer;