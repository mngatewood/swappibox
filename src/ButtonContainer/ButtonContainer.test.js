import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount, render, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import createRouterContext from 'react-router-test-context';
import PropTypes from 'prop-types';
import ButtonContainer from './ButtonContainer';

describe('ButtonContainer', () => {

  configure({ adapter: new Adapter() });
  let wrapper;

  beforeEach(() => {
    wrapper = mount(<ButtonContainer />);
  });

  // it('renders without crashing', () => {
  //   const div = document.createElement('div');
  //   ReactDOM.render(<ButtonContainer />, div);
  //   ReactDOM.unmountComponentAtNode(div);
  // });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })

});