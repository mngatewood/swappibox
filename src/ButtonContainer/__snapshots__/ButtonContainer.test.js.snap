// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonContainer should match the snapshot 1`] = `undefined`;
